tasks/copy_to_glacier/schemas/config.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/copy_to_files/schemas/config.json",
+  "title": "copy_to_files Lambda Config",
+  "description": "The config for the copy_to_files Lambda.",
+  "type": "object",
+  "required": ["collection"],
+  "properties": {
+    "collection": {
+      "description": "The collection from AWS.",
```

Comment:
This collection is not from AWS but comes from Cumulus.

Reply:
Fixed